src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/UnsupportedTypeConverter.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class UnsupportedTypeConverter<T> : JsonConverter<T>
+    {
+        private static readonly string s_exceptionMessage = SR.Format(SR.SerializeTypeInstanceNotSupported, typeof(T).FullName);
```

Comment:
Do we cache resource strings elsewhere? IMO we should just re-create the instance every time and save a bit of private memory.